### ISSUE TITLE
Does this support DataChannel?

### ISSUE THREAD

**Issue by zhBugFly** [webrtc-uwp/webrtc-uwp-sdk#167]
I have established a vedio connection between HL and Uwp application running on the windows. Then I try to create a datachannelbased on the connection to send a string message. The datachannel worked well and I can get the message. But when I hangup the vedio, both applicaions running on the HL or PC crashed.  Icannot find out where the crash occured.

**Comment by jamescadd** [comment]
Yes DataChannel is supported.  If you run this as a debug build with VS attached and check the debug output you should see some info about the crash, could you post this?
